components/server/src/orgs/organization-service.ts

```diff
@@ -549,6 +550,17 @@ export class OrganizationService {
             }
         }
 
+        if (settings.onboardingSettings?.internalLink) {
+            if (
+                !isURL(settings.onboardingSettings.internalLink ?? "", {
+                    require_protocol: true,
+                    host_blacklist: ["localhost", "127.0.0.1"],
```

Comment:
Do we need to exclude IPv6 localhost (`::1`) as well?

Reply:
Sounds good, will add. We don't have to be too strict here though, I just wanted to add URLs that you probably really don't want to add and might try to accidentally.

Reply:
Ah, correct. :+1: